pontoon/base/templates/home.html

```diff
@@ -0,0 +1,160 @@
+              </div>
+            </div>
+            <div class="card-features flex vertical-align-center">
+              <div class="card-image"><img src="/static/img/Homepage/feature-4.svg"></img>
+              </div>
+              <div class="card-text content-style colored-content-style">Mass actions for quick translation and review tasks.
+              </div>
+            </div>
+            <div class="card-features flex vertical-align-center">
+              <div class="card-image"><img src="/static/img/Homepage/feature-5.svg"></img>
+              </div>
+              <div class="card-text content-style colored-content-style">Automated bi-di synchronization with version control systems.
+              </div>
+            </div>
+            <div class="card-features flex vertical-align-center">
+              <div class="card-image"><img src="/static/img/Homepage/feature-6.svg"></img>
+              </div>
+              <div class="card-text content-style colored-content-style">In-context localization.
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Section-5: In-context localization -->
+    <section class="snaps section-5 white-background" data-section-name="section-5">
+      <div class="container vertical-align-center container--full-height">
+        <div class="in-context flex-col-6 flex-direction-col neg-margin--large">
+          <div class="title-style colored-title-style align-left">One more thing.</div>
+          <div class="content-style colored-content-style">One of the coolest features of Pontoon is localizing web content in-place, with context and spatial limitations right in front of you. The moment you submit translation, it replaces the original text in the web page, making you the first proofreader and tester. <br/><br/>
+          We made a special demo page for you to understand it better.<br/></div>
+          <a class="homepage-button primary-button-cta" href="localhost:8000/in-context/">
+            <div>Give it a try!</div>
+          </a>
+        </div>
+        <div  class= "flex-col-6 align-right"><img class="section-side-image-1 neg-margin--large" src="/static/img/Homepage/section-5.svg"></img></div>
+      </div>
+    </section>
+
+    <!-- Section-6: Final -->
+    <section class="snaps section-6" data-section-name="section-6">
+      <div class="footer vertical-align-center flex-direction-col">
+        <div class="title-style">Join us.</div>
+        <p class="content-style">Now that you know what Pontoon is, come on in.</p>
+        <a class="homepage-button primary-button-cta" href="localhost:8000/teams/">
+          <div>Localize Mozilla</div>
+        </a>
+        <p class="content-style">Want to learn more?</p>
+        <a class="homepage-button secondary-button-cta" href="localhost:8000/demo/">
+          <div>Take a tour!</div>
```

Comment:
Make sure you follow the latest proposal:
https://ibb.co/hAZgK8

The CTAs in the Final section must match the CTAs in the Introduction.